Iceberg.package/IceRepository.class/instance/origin.st

```diff
@@ -1,3 +1,9 @@
 accessing remotes
 origin
-	^ self backend origin
\ No newline at end of file
+	^ [ self backend origin ]
+		on: LGit_GIT_ENOTFOUND 
+		do: [ :error | "if local repository is deleted, LGit_GIT_ENOTFOUND will be thrown. We already know the origin"
+				{ pullRemote . pushRemote }
```

Comment:
this needs to be `self pushRemote. self pullRemote`, not variables directly, because they can be nil (not initialised) and tnen the rest of the verification will have a DNU :P